dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/build.gradle

```diff
@@ -11,12 +11,21 @@
 
 configurations {
 	testNG { transitive = false; }
+  arquillian
+  wlp
 }
 
 dependencies {
 	testNG 'org.testng:testng:6.14.3',
 	       'com.beust:jcommander:1.78',
 	       'org.apache-extras.beanshell:bsh:2.0b6'
+
+  arquillian 'io.openliberty.arquillian:arquillian-liberty-support-jakarta:2.0.1'
+
+  //TODO configure WLP dependency, Examples (may not be valid):
+  //wlp 'io.openliberty.beta:openliberty-jakartaee10:22.0.0.2-beta@zip'
+  //wlp 'io.openliberty.beta:openliberty-runtime:22.0.0.2-beta@zip'    
+  //wlp 'io.openliberty:openliberty-jakartaee9:22.0.0.1@zip'
```

Comment:
Could you switch to 22.0.0.3-beta which is the first beta that will pass the Concurrency 3.0 TCK?
Also, let's leave out the jakartaee9 example given that the first EE spec level that we are automating the TCK for is EE 10